### PR TITLE
Adding STARTUPINFOEXA/STARTUPINFOEXA and two constants for SetHandleInformation

### PR DESCRIPTION
**Pull request by yying**

### CODE REVIEW

File: src/um/winbase.rs [hunk 1]
```diff
@@ -639,6 +639,8 @@ pub const SEM_FAILCRITICALERRORS: UINT = 0x0001;
 pub const SEM_NOGPFAULTERRORBOX: UINT = 0x0002;
 pub const SEM_NOALIGNMENTFAULTEXCEPT: UINT = 0x0004;
 pub const SEM_NOOPENFILEERRORBOX: UINT = 0x8000;
+pub const HANDLE_FLAG_INHERIT: DWORD = 0x00000001;
+pub const HANDLE_FLAG_PROTECT_FROM_CLOSE: DWORD = 0x00000002;
```
Comment (retep998) [inline src/um/winbase.rs]: These constants don't belong here. Rather they should go in between `SetFileShortNameW` and `LoadModule`.

File: src/um/winbase.rs [hunk 2]
```diff
@@ -999,6 +1001,16 @@ pub const STARTF_TITLEISLINKNAME: DWORD = 0x00000800;
 pub const STARTF_TITLEISAPPID: DWORD = 0x00001000;
 pub const STARTF_PREVENTPINNING: DWORD = 0x00002000;
 pub const STARTF_UNTRUSTEDSOURCE: DWORD = 0x00008000;
+STRUCT!{struct STARTUPINFOEXA {
+    StartupInfo: STARTUPINFOA,
+    lpAttributeList: PPROC_THREAD_ATTRIBUTE_LIST,
```
Comment (retep998) [inline src/um/winbase.rs]: This should be `LPPROC_THREAD_ATTRIBUTE_LIST` instead of `PPROC_THREAD_ATTRIBUTE_LIST`.

File: src/um/winbase.rs [hunk 3]
```diff
@@ -999,6 +1001,16 @@ pub const STARTF_TITLEISLINKNAME: DWORD = 0x00000800;
...
+}}
+pub type LPSTARTUPINFOEXA = *mut STARTUPINFOEXA;
+STRUCT!{struct STARTUPINFOEXW {
+    StartupInfo: STARTUPINFOW,
+    lpAttributeList: PPROC_THREAD_ATTRIBUTE_LIST,
```
Comment (retep998) [inline src/um/winbase.rs]: Same here.

**Review by retep998 — CHANGES_REQUESTED** [review]
A few tweaks and this should be good to go.

File: src/um/winbase.rs (hunk 1 above)
Comment (yying) [reply src/um/winbase.rs]: Done. I ended the prior `extern` block, inserted the new consts, and continued with a new `extern` block for `LoadModule`.

File: src/um/winbase.rs (hunk 2 above)
Comment (yying) [reply src/um/winbase.rs]: Done.

File: src/um/winbase.rs (hunk 3 above)
Comment (yying) [reply src/um/winbase.rs]: Done.

**Review by retep998 — APPROVED** [review]

**Pull request merged**